python3Pyside2QML/.vscode/tasks.json

```diff
@@ -177,8 +177,25 @@
             "icon": {
                 "id": "layers",
                 "color": "terminal.ansiCyan"
+            },
+            "dependsOn": [
+                "host-upgrade-pip"
+            ]
+        },
+        {
+            "label": "host-upgrade-pip",
+            "detail": "hide",
```

Comment:
We are shipping now the new way, instead of the `"detail": "hide"` use `"hide": true`